### DISCUSSION TITLE
Should I use Sigma Rules with status: test?

### DISCUSSION THREAD

**Discussion by Carlos-mb** [Security-Onion-Solutions/securityonion#13532] [Unsupported Versions] [answered]
Hi all

I have seen that there are 1582 Sigma rules and 1420 have "status: test".

It should be great to know your opinion about if this is right and if I should I use these rules in a production environment. 

What do you think?

**Comment by defensivedepth** [comment, marked as answer]
Check out the Sigma documentation which gives a rundown on various aspects of a Sigma rule including `status` - https://github.com/SigmaHQ/sigma/blob/r2024-07-17/Releases.md
